core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -711,6 +711,15 @@ public NativeImageInvokerInfo build() {
                     }
                 }
 
+                // only available in GraalVM 22.3.0 and better.
+                if (graalVMVersion.compareTo(GraalVM.Version.VERSION_22_3_0) >= 0) {
```

Comment:
```suggestion
                if (graalVMVersion.isNewerThan(GraalVM.Version.VERSION_22_2_0)) {
```

In the rest of the code we use `isOlderThan` and `isNewerThan` instead of `compareTo` for such checks. Might want to change it for consistency.

Reply:
Sure, but it more clearly states the intent, doesn't it? `graalVMVersion.isNewerThan(GraalVM.Version.VERSION_22_2_0)` would be true for a hypothetical `22.2.1` release.

Reply:
True. Maybe we need some more methods :)